packages/modules/src/networkAnalysisDataVis/components/statistics/GraphStatsPanel.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+
+import { TableCell as MuiTableCell, TableCellProps } from '@material-ui/core'
+import Accordion from '@material-ui/core/Accordion'
+import AccordionSummary from '@material-ui/core/AccordionSummary'
+import AccordionDetails from '@material-ui/core/AccordionDetails'
+import Typography from '@material-ui/core/Typography'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableRow from '@material-ui/core/TableRow'
+import Divider from '@material-ui/core/Divider'
+
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+
+import { OutputValues } from '../../structure'
+import useStyles from './GraphStatsPanel.styles'
+
+export interface GraphStatsPanelProps {
+  graphStats: OutputValues['graphStats']
+}
+
+const TableCell = ({ children }: TableCellProps): JSX.Element => {
+  const classes = useStyles()
+  return <MuiTableCell classes={{ root: classes.tableCellRoot }}>{children}</MuiTableCell>
+}
+
+export const GraphStatsPanel = ({ graphStats }: GraphStatsPanelProps): JSX.Element => {
+  const classes = useStyles()
+
+  const [expanded, setExpanded] = useState<boolean>(false)
+
+  return (
+    <div className={classes.statContainer}>
+      <Accordion expanded={expanded} onChange={() => setExpanded(prevStatus => !prevStatus)}>
+        <AccordionSummary expandIcon={<ExpandMoreIcon />} aria-controls="panel1d-content" id="panel1d-header">
```

Comment:
The ID is not referenced anywhere?

Reply:
Yes, that's another unnecessary attribute from the MUI docs that I just forgot to remove.